providers/shared/components/correspondent/id.ftl

```diff
@@ -0,0 +1,36 @@
+[#ftl]
+
+[@addComponentDeployment
+    type=CORRESPONDENT_COMPONENT_TYPE
+    defaultGroup="solution"
+    defaultPriority=70
+/]
+
+[@addComponent
+    type=CORRESPONDENT_COMPONENT_TYPE
+    properties=
+        [
+            {
+                "Type"  : "Description",
+                "Value" : "An outbound and inbound marketing communications service such as AWS Pinpoint"
+            }
+        ]
+    attributes=
+        [
+            {
+                "Names" : "Profiles",
+                "Children" : [
+                    {
+                        "Names" : "Logging",
```

Comment:
Doesn't look this is used at the moment, might be worth removing the profile configuration for now

Reply:
Profiles attribute has been removed.